src/components/profile.js

```diff
@@ -1,19 +1,24 @@
 import React, { Component } from "react"; // eslint-disable-line no-unused-vars
 import { Link } from "react-router";
 import Gravatar from "react-gravatar";
+import Item from "./profile-item";
+import ProfileEdit from "./profile-edit";
+import ProfileLink from "./profile-link";
 
 class Profile extends Component {
 	constructor() {
 		super();
 
 		this.state = {
-			profile: {}
+			profile: {},
+			phone: ""
 		};
 	}
 
 	componentWillMount() {
 		this.setState( {
-			profile: this.props.profiles[ 0 ]
+			profile: this.props.profiles,
+			phone: this.props.profiles.phone
```

Comment:
Again, I'm not sure the value of setting these into `state` rather than using `this.props...` directly.